Core/Core/Inbox/Addressbook/View/AddressbookRecipientView.swift

```diff
@@ -0,0 +1,111 @@
+//
+// This file is part of Canvas.
+// Copyright (C) 2023-present  Instructure, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+//
+
+import SwiftUI
+
+public struct AddressbookRecipientView: View {
+    @ObservedObject private var viewModel: AddressbookRecipientViewModel
+    @Environment(\.viewController) private var controller
+
+    init(model: AddressbookRecipientViewModel) {
+        self.viewModel = model
+    }
+
+    public var body: some View {
+        ScrollView {
+            peopleView
+        }
+        .searchable(
+            text: Binding { viewModel.searchText.value } set: { viewModel.searchText.send($0) },
+            placement: .navigationBarDrawer(displayMode: .always)
+        )
+        .background(Color.backgroundLightest)
+        .navigationTitle(viewModel.title)
+    }
+
+    private var separator: some View {
+        Color.borderMedium
+            .frame(height: 0.5)
+    }
+
+    private var peopleView: some View {
+        VStack(alignment: .leading, spacing: 0) {
+            separator
```

Comment:
I think this separator is not needed here. We have a default separator below the search bar and with this one also added the two looks brighter than the others.
![IMG_0194](https://github.com/instructure/canvas-ios/assets/72396990/0f00ed93-21f5-483c-87f6-9dbee4064cff)